content/en/docs/develop/core-modules/governance.md

```diff
@@ -246,19 +420,31 @@ for finishedProposalID in GetAllFinishedProposalIDs(block.Time)
 The subspace for the Governance module is `gov`.
 
 ```go
+// DepositParams defines the params for deposits on governance proposals.
 type DepositParams struct {
-	MinDeposit       sdk.Coins     `json:"min_deposit,omitempty" yaml:"min_deposit,omitempty"`
-	MaxDepositPeriod time.Duration `json:"max_deposit_period,omitempty" yaml:"max_deposit_period,omitempty"` //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
+	//  Minimum deposit for a proposal to enter voting period.
+	MinDeposit github_com_cosmos_cosmos_sdk_types.Coins `protobuf:"bytes,1,rep,name=min_deposit,json=minDeposit,proto3,castrepeated=github.com/cosmos/cosmos-sdk/types.Coins" json:"min_deposit,omitempty" yaml:"min_deposit"`
+	//  Maximum period for Atom holders to deposit on a proposal. Initial value: 2
```

Comment:
denom update is needed

Reply:
https://github.com/xpladev/docs/pull/13/commits/6155517bc6257cdd88fa7db09b9645b643c5956d